tests/driver_esp_eth/Makefile.ci

```diff
@@ -0,0 +1,28 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
Are you should this is needed? Wouldn't `FEATURES_REQUIRED += arch_esp32` rule out all ARM and AVR boards anyway?

Reply:
True

Reply:
I removed the file